### PR TITLE
update text

### PR DESCRIPTION
**Pull request by jameshall9327**
update provide to provides

Goal:

Ticket(s): #

- [ ] Backport to LTR documentation is requested

### CODE REVIEW

File: docs/gentle_gis_introduction/vector_data.rst [hunk 1]
```diff
@@ -13,7 +13,7 @@ Vector Data
 Overview
 ========
 
-**Vector** data provide a way to represent real world **features** within the GIS
+**Vector** data provides a way to represent real world **features** within the GIS
```
Comment (DelazJ) [inline docs/gentle_gis_introduction/vector_data.rst:16]: Technically speaking, the word "data" is plural: https://duckduckgo.com/?q=is+data+plural+or+singular&t=h_

Comment (jameshall9327) [reply docs/gentle_gis_introduction/vector_data.rst:16]: not data the word provide

Comment (DelazJ) [reply docs/gentle_gis_introduction/vector_data.rst:16]: I meant: if "data" is plural, then the verb (to provide) should also be plural (provide), not singular (provides). No?

Comment (timlinux) [reply docs/gentle_gis_introduction/vector_data.rst:16]: Agreed with @DelazJ data is plural so provide should be plural too.

**Pull request closed without merging**